.github/workflows/lint.yaml

```
@@ -217,9 +217,11 @@ jobs:
         uses: bufbuild/buf-breaking-action@c57b3d842a5c3f3b454756ef65305a50a587c5ba # v1.1.4
         with:
           against: 'https://github.com/${GITHUB_REPOSITORY}.git#branch=${{ github.event.pull_request.base.sha || github.event.merge_group.base_sha }}'
+      # Release branches must not add new fields to protos with respect to
+      # master unless they reserve the field on master.
```

I think I changed my stance compared to my [original comment](https://github.com/gravitational/teleport/pull/30649/files#diff-4b122024a3a28ded65da76a2f1bface1f3a27328374438d1298d25585fe0603bL51) and I think we should discourage removing fields altogether: 

```suggestion
      # Release branches must not add new fields to protos with respect to
      # master, new fields should be added to master first.
```